hypervisor/include/arch/x86/mmu.h

```diff
@@ -117,15 +117,15 @@ void mmu_modify_or_del(uint64_t *pml4_page, uint64_t vaddr_base, uint64_t size,
 /**
  * @brief EPT and VPID capability checking
  *
- * @return 0 - on success
- * @return -ENODEV - Don't support EPT or VPID capability
+ * @retval 0 on success
+ * @retval -ENODEV Don't support EPT or VPID capability
  */
 int check_vmx_mmu_cap(void);
 /**
  * @brief VPID allocation
  *
- * @return 0 - VPID overflow
- * @return more than 0 - the valid VPID
+ * @retval 0 VPID overflow
+ * @retval positive the valid VPID
```

Comment:
How about changing positive to int? (that keyword is formatted as if it is a parameter)
```suggestion
 * @retval int the valid VPID
```

Reply:
`int` is broader than `positive` and may be confusing because the return value has type uint16_t while `int` can be negative. How about using `>0` here, like replacing `negative` with `<0` below?